spark/src/test/scala/org/apache/spark/sql/delta/SchemaValidationSuite.scala

```diff
@@ -68,12 +69,7 @@ class SchemaValidationSuite
     }
   }
 
-  def cloneSession(spark: SparkSession): SparkSession = {
-    val cloneMethod = classOf[SparkSession].getDeclaredMethod("cloneSession")
-    cloneMethod.setAccessible(true)
-    val clonedSession = cloneMethod.invoke(spark).asInstanceOf[SparkSession]
-    clonedSession
-  }
+  def cloneSession(spark: SparkSession): SparkSession = spark.cloneSession()
```

Comment:
I am not sure why we need these gymnastics. As long as the test is defined in org.apache.spark.. you can call `SparkSession.cloneSession()` directly.